.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -4163,7 +4150,7 @@ stages:
           artifact: ssi-artifacts
 
 - stage: store_serverless_artifacts
-  condition: and(succeeded(), eq(variables['isBenchmarksOnlyBuild'], 'False'), eq(variables.isMainRepository, true))
+  condition: and(succeeded(), eq(variables.isMainRepository, true))
```

Comment:
we can probably delete all the `isMainRepository` stuff too now IMO. This is all left over from ages ago, before we merged the debugger. I don't think we need it now

Reply:
Removed 👍